libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

```diff
@@ -151,7 +151,10 @@ export async function getDynamicSchema(
 
       switch (dynamicState?.extension?.builtInOperation) {
         case 'getVariableSchema':
-          schema = { type: getSwaggerTypeFromVariableType(operationParameters['type']?.toLowerCase() ?? 'boolean') };
+          schema = {
+            type: getSwaggerTypeFromVariableType(operationParameters['type']?.toLowerCase() ?? 'boolean'),
+            enum: getSwaggerEnumFromVariableType(operationParameters['type']?.toLowerCase() ?? 'boolean'),
```

Comment:
Why is this change needed?

Reply:
This is a separate Fix, #2023 I changed it so it would get the enum values, but this was also getting caught on initialize variable boolean to have the options yes/no/ ' ', instead of true/false